core/src/main/java/cz/xtf/core/helm/ConfiguredVersionHelmBinaryPathResolver.java

```diff
@@ -46,7 +46,7 @@ private String unpackHelmClientArchive(final Path archivePath, final boolean del
         try {
             List<String> args = Stream
                     .of("tar", "-xf", archivePath.toAbsolutePath().toString(), "-C",
-                            getProjectHelmDir().toAbsolutePath().toString())
+                            getProjectHelmDir().toAbsolutePath().toString(), "--no-same-owner")
```

Comment:
can you please add a comment on why `--no-same-owner` is added?

Reply:
Explanation comment added.